docs/source/migration.mdx

```diff
@@ -321,7 +321,6 @@ This plugin installs the [unmaintained](https://github.com/graphql/graphql-playg
 | `ApolloServerPluginInlineTrace` | `@apollo/server/plugin/inlineTrace` |
 | `ApolloServerPluginInlineTraceDisabled` | `@apollo/server/plugin/disabled` |
 | `ApolloServerPluginLandingPageDisabled` | `@apollo/server/plugin/disabled` |
-| `ApolloServerPluginLandingPageGraphQLPlayground` | `@apollo/server-plugin-landing-page-graphql-playground` |
```

Comment:
so just to confirm, we're leaving the paragraphs above the table, just removing the entry? sounds good.

Reply:
Yes. The entry in this table is a bit weird / out of place as it is, since Apollo Server does not export this package.